homeassistant/components/alexa/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Alexa",
   "documentation": "https://www.home-assistant.io/integrations/alexa",
   "requirements": [],
-  "dependencies": ["http"],
+  "dependencies": ["http", "image_processing"],
```

Comment:
This should be in `after_dependencies`. 

Related to #29913 

However `cloud` does not rely on Alexa so this would actually not get installed.